sdk/cosmosdb/cosmos/src/queryExecutionContext/orderByDocumentProducerComparator.ts

```diff
@@ -35,9 +35,9 @@ const TYPEORDCOMPARATOR: {
 
 /** @hidden */
 export class OrderByDocumentProducerComparator {
-  constructor(public sortOrder: string[]) {} // TODO: This should be an enum
+  constructor(public sortOrder: string[]) { } // TODO: This should be an enum
 
-  public targetPartitionKeyRangeDocProdComparator(
+  private targetPartitionKeyRangeDocProdComparator(
```

Comment:
Is this a breaking change to switch from `public` to `private`, or is this class not exposed to customers?

Reply:
Correct. It is an internal class